### PR TITLE
Adds prometheus metrics endpoint

### PR DESCRIPTION
**Pull request by StrongestNumber9**

### CODE REVIEW

**Comment by StrongestNumber9** [comment]
Will include the DropWizard stats on the same PR as further commit as they are linked together

**Comment by StrongestNumber9** [comment]
Implemented some of the metrics, and they require plenty of review especially with the errors/reconnects/connections part

File: src/main/java/com/teragrep/k8s_01/RelpOutput.java [hunk 1]
```diff
@@ -142,14 +158,30 @@ public void send(SyslogMessage syslogMessage) {
...
         }
     }
 
     public int getId() {
         return id;
     }
+
+    private void loadMetrics() {
+        // All registered through PrometheusMetrics
+        MetricRegistry metricRegistry = SharedMetricRegistries.getOrCreate("default");
```
Comment (kortemik) [inline src/main/java/com/teragrep/k8s_01/RelpOutput.java:177]: pass rather as an object in constructor than singleton getter

**Review by kortemik — CHANGES_REQUESTED** [review]

**Comment by StrongestNumber9** [comment]
Requested fixes done, self approving for now

**Pull request merged**